app/views/layouts/_flash_msg.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="flash_messages">
```

Comment:
Instead of this large flash case, try something like this

https://github.com/osulp/Room-Reservation/blob/9c00bfbde596f50d7c72bcda73d4a2ee99609e92/app/views/application/_flashes.html.erb